dae-alright/src/Redux/constants/action-type.js

```diff
@@ -0,0 +1,6 @@
+
+
+const SIMPLE_ACTION = "SIMPLE_ACTION";
+
+
+export default SIMPLE_ACTION
```

Comment:
you should export each constant or make your `default` export to be
```
export default {
    SIMPLE_ACTION
}
```

Reply:
Noted